src/app/preferences/components/Routes/Routes.tsx

```diff
@@ -76,7 +80,10 @@ const Routes = (): JSX.Element => {
       />
       <Route
         element={
-          <PageContent sidePanelContent={null} sidePanelTitle={null}>
+          <PageContent
+            sidePanelContent={sidePanelContent && <DeleteSSHKey />}
+            sidePanelTitle="Delete SSH key"
+          >
```

Comment:
This is no longer necessary.
```suggestion
          <PageContent sidePanelContent={null} sidePanelTitle={null}>
```